hail/python/hailtop/batch/job.py

```diff
@@ -348,6 +348,17 @@ def storage(self, storage: Union[str, int]) -> 'Job':
         *G*, *Gi*, *T*, *Ti*, *P*, and *Pi*. Omitting a suffix means
         the value is in bytes.
 
+        For the :class:`.ServiceBackend`, by default every job gets 5 Gi
+        per core requested for the root file system `/` with a maximum of 5 Gi.
```

Comment:
I think this is less confusing as: "jobs requesting one or more cores receive 5 GiB of storage for the root file system `/`. Jobs requesting a fraction of a core receive the same fraction of 5 GiB of storage."